plugin-build/src/main/kotlin/io/sentry/android/gradle/AndroidComponentsConfig.kt

```diff
@@ -243,6 +246,18 @@ fun AndroidComponentsExtension<*, *, *>.configure(
         }
       }
     }
+    if (extension.sizeAnalysis.enabled.get() == true) {
+      val sentryTelemetryProvider =
```

Comment:
not sure if it is okay to re-initialize this here. it is also initialized on line 70 per variant.